packages/shylu/shylu_node/tacho/unit-test/CMakeLists.txt

```diff
@@ -25,6 +25,16 @@ TRIBITS_ADD_LIBRARY(
   NO_INSTALL_LIB_OR_HEADERS
 )
 
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Tacho_TestSolver.x
+  NOEXESUFFIX
+  NOEXEPREFIX
```

Comment:
I haven't used this before, just adding a note from the TriBITS documentation for reference:
```
NOEXEPREFIX

    If passed in, then ${PACKAGE_NAME}_ is not added the beginning of the executable name
```